src/DependencyInjection/CompilerPass/LDLAbstractCompilerPass.php

```diff
@@ -4,10 +4,40 @@
 
 namespace LDL\DependencyInjection\CompilerPass;
 
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
 abstract class LDLAbstractCompilerPass implements LDLCompilerPassInterface
 {
+    private static $defaultPassType;
+
+    private static $defaultPriority;
+
     final public function __construct()
     {
 
     }
+
+    private static function setDefaultArguments() : void
+    {
+        if(null !== self::$defaultPassType){
+            return;
+        }
+
+        $rm = new \ReflectionMethod(ContainerBuilder::class, 'addCompilerPass');
+
```

Comment:
Si no existe ese metodo (habria que ver que devuelve esto), hay que tirar una excepcion violenta aca

Reply:
throw new \RuntimeException('Container builder class does not has any method named addCompilerPass, this is a critical error, please report the error in github by creating a new issue');